helm/hpcc/values.schema.json

```diff
@@ -1411,6 +1419,36 @@
         }
       ]
     },
+    "dafilesrv": {
+      "type": "object",
+      "required": [ "name" ],
```

Comment:
should service be required? (the rest of the helm code assumes it exists)

Reply:
yes, I'll make it required